app/app.go

```diff
@@ -360,6 +366,9 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	)
 	app.AuthzKeeper = authzKeeper
 
+	groupConfig := group.DefaultConfig()
```

Comment:
group module [default-config](https://github.com/cosmos/cosmos-sdk/blob/main/x/group/config.go#L17-L18).

Reply:
Defaults look good to me.